lib/cc_plugin_imos/cc_plugin_imos/tests/data/imos_test_variable.cdl

```diff
@@ -57,5 +54,5 @@ data:
 
  bad2_qc = -127 ;
 
- bad3_qc = -127 ;
+ new_qc = -127 ;
 }
```

Comment:
Yes, it would probably make it easier to update the tests in the future. For now I think this is ok. I can create separate cdl files in a future "clean-up" PR.